test/dlc_tests/sanity/test_security_check.py

```diff
@@ -35,8 +36,9 @@ def test_security(image):
 
 
 @pytest.mark.model("N/A")
+@pytest.mark.canary("Run ECR Scan test regularly on production images")
```

Comment:
Do we want to run it as a regular canary test? I thought we discussed that we wanted to only run it on a selected 30-min time window per day.

Reply:
The reason we wanted to restrict the other tests was because of resource limitations. There are no similar resource limitations here, because the ECR Scan Tool itself already runs only once a day on a single image. We cannot run it more frequently than that.